api/src/paths/dwc/dataset/list.test

```diff
@@ -0,0 +1,308 @@
+
```

Comment:
This file is missing its `.ts` extension

Reply:
I did this intentionally, as the tests aren't finished and I wanted the rest of the test suite to run